docker/release/config/.vscode/c_cpp_properties.json

```diff
@@ -0,0 +1,19 @@
+{
+    "configurations": [
```

Comment:
This configuration still leads to IntelliSense errors when using cudaq::adjoint (stating there is no template for these types). However, I suspect we cannot fix that without changing how the language server works, since we would likely need to add to the design time build.